flink-ml-iteration/src/main/java/org/apache/flink/ml/iteration/IterationBody.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.iteration;
+
+/** The builder of the subgraph that will be executed inside the iteration. */
+public interface IterationBody {
+
+    /**
+     * This method creates the graph for the iteration body. See {@link Iterations} for how the
```

Comment:
It seems that the Java doc is different from FLIP-176. It will be nice if we can keep Java doc here consistent with the FLIP.

Among other things, can we keep the requirement that "the number of feedback variable streams returned by this method must equal the number of variable streams given to this method."

Reply:
@lindong28 I think this sentence should indeed be kept. Also refer to the following comments, I also agree we have a overall revise to the javadoc in this class.